src/index.html

```diff
@@ -4,9 +4,360 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/style.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap" rel="stylesheet"
+    crossorigin="true">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="catalog">
+
+      <header class="header">
+        <a href="#" class="navbar-logo">
+          <img
+            class="logo_img"
+            src="images/logo.png"
+            alt="logo"
+          >
+        </a>
+        <nav class="navbar">
+          <ul class="navbar-list">
+            <li class="navbar-item">
+              <a 
+                href="#" 
+                class="is-active"
+              >
+                APPLE
+              </a>
+            </li>
+            <li class="navbar-item">
+              <a href="#" class="navbar-link">SAMSUNG</a>
```

Comment:
```suggestion
                class="navbar-link is-active"
              >
                APPLE
              </a>
            </li>
            <li class="navbar-item">
              <a href="#" class="navbar-link">SAMSUNG</a>
```

Reply:
ok, but  that's what I got
<img width="1440" alt="Screenshot 2023-10-23 at 17 29 40" src="https://github.com/mate-academy/layout_catalog/assets/109960482/6b2ae94d-6fbf-4659-8f54-794fe6fe6a63">